src/content/docs/data-apis/ingest-apis/metric-api/cardinality-management.mdx

```diff
@@ -73,81 +60,95 @@ You can view a deeper layer of granularity in your metrics by clicking on any me
+The <DNT>**Usage over time**</DNT> chart displays the metric cardinality usage compared the limit of that metric. You can adjust the time picker to view different time periods of this individual metric's cardinality data.
 
 <Callout variant="tip">
-  For any metric that has exceeded its cardinality limit (usually 100,000) New Relic will still attempt to populate information on the metric, but it will be slightly slower since it uses RAW data instead of using aggregated data. 
+    If a metric exceeds its cardinality limit, New Relic still populates information for the metric, but it may be slightly slower as it uses raw data instead of aggregated data.
 </Callout>
 
 ### Attributes table [#attributes-table]
 
-Under the timeseries, a table shows the attributes that are powering the metric cardinality. Note that the numbers usually don’t add up to 100% because cardinality is calculated as a combination of several attributes. However, it is a good indicator of what your highest contributing attributes are. 
-
-<img
-    style={{ align: 'left',maxWidth: '90%' }}
-    title="Screenshot showing attributes behind metrics"
-    alt="Screenshot showing attributes behind metrics"
-    src="/images/apis-and-data_screenshot-crop_metric-attributes.webp"
-/>
-
-This information is only for the last day (since UTC midnight), which is when cardinality resets (both account and metric). 
-
-You can also view specific information about an attribute, which can help you make decisions about whether or not that attribute is useful. You can see a sample of the attribute values by clicking the menu on the right side of the attribute row.
-
+The <DNT>**Attributes**</DNT> table has the list of top 20 attributes that majorly contribute to the metric's cardinality. To view the values of a specific attribute, click the options menu on the right side of the attribute row and select <DNT>**View attribute values**</DNT>.
 
 ## Account pruning rules [#account-pruning-rules]
 
-A common way to keep account cardinality under its limit is to implement pruning rules for metrics and attributes you don't consider important. 
+A common method to maintain account cardinality within its limit is to implement pruning rules for non-critical attributes, which removes unnecessary data.
 
 ### View account pruning rules [#view-pruning-rules]
 
-You can view pruning rules by clicking into a metric and then clicking **View Pruning Rules**. This will bring up a scrollable list of all pruning rules associated with an account. You can also delete pruning rules associated with an account in this view:
-
+To view account pruning rules:
+1. Go to the **Cardinality Management** page of the account.
+2. From the **Metrics** table, select <DNT>**View Pruning Rules**</DNT> to get a scrollable list of all pruning rules associated with the account.
```

Comment:
To gets a scrollable....

Reply:
Reframed the sentence as following:
From the **Metrics** table, select <DNT>**View Pruning Rules**</DNT>. A scrollable list of all pruning rules associated with the account appears.